pkg/apiserver/handler.go

```diff
@@ -284,6 +285,15 @@ func handle(ginCtx *gin.Context, handler HandlerWithoutInput, input interface{},
 		return
 	}
 
+	var notFound db_repo.RecordNotFoundError
```

Comment:
this is one step to deep down in our abstractions..  
instead it should be handled in the `pkg/apiserver/crud/read.go` handler..  
`apiserver/crud` is connected to the `db_repo` package while the underlying `apiserver` package is not. we should keep it this way.